packages/pass-style/test/safe-promise.test.js

```diff
@@ -39,9 +40,7 @@ test('safe promise loophole', t => {
     defineProperty(p3, toStringTag, {
       value: 3,
     });
-    t.throws(() => passStyleOf(harden(p3)), {
-      message: 'Own @@toStringTag value must be a string: 3',
-    });
+    t.is(passStyleOf(harden(p3)), 'promise');
```

Comment:
The change to this test case demonstrates the loosening of the `@@toStringTag` tests